ui/App.tsx

```diff
@@ -6,8 +6,10 @@ import {
   Route,
   Switch,
 } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
```

Comment:
Ah hah more toast. Cool. Well its all in one place if we need to reconcile Snackbar and Toast 👌